src/BugsnagUnity/Payload/Report.cs

```diff
@@ -27,11 +28,51 @@ internal Report(IConfiguration configuration, Event @event)
       this.AddToPayload("events", new[] { Event });
     }
 
-    public Event Event { get; }
+    Event Event { get; }
+
+    internal HandledState OriginalSeverity => Event.OriginalSeverity;
 
     public void Ignore()
     {
       Ignored = true;
     }
+
+    public Metadata Metadata => Event.Metadata;
```

Comment:
Do we need inline docs for any of these properties, or is that being addressed separately?

Reply:
Good idea, I'll add them there as well. I also need to update the docs site which I have a note to do